pluto/src/color/core/color.ts

```diff
@@ -193,6 +193,17 @@ export class Color {
     return Number((0.2126 * r + 0.7152 * g + 0.0722 * b).toFixed(3));
   }
 
+  /**
+   * @returns the grayness of the color, between 0 and 1.
+   */
+  get grayness(): number {
```

Comment:
I just am not sure that `grayness` makes sense here, since it takes into account the brightness as well. You could probably get a good measure of grayness with a formula like this:

avg = (r + g + b)/3
grayness = sqrt((r-avg)^2 + (g-avg)^2 + (b-avg)^2)

Reply:
Problem is that this makes yellow super gray

Reply:
Updated